kfdefs/overlays/moc/smaug/opf-trino-stage/vpa/vpa-custom-resource.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  namespace: opf-trino-stage
+  name: vpa-trino-stage
+  labels:
+    app: vpa-trino-stage-v1
+spec:
+  resourcePolicy:
+    containerPolicies:
+      - containerName: trino-worker
+        maxAllowed:
+          cpu: 3000m
+          memory: 3Gi
+        minAllowed:
```

Comment:
Any particular reason you don't have a minAllowed set for memory?

Reply:
When we observed the historical data, the memory utilization hardly dropped below 500MB. So, we thought the minAllowed for Memory is not required as VPA by default sets the minAllowed as 250MB which is fine in our case. So we did not specify any value for Memory.

Reply:
250mb is not the minAllowed value https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go#L161 by default. However, from our observations, it never recommends less than 250mb.